packages/server/src/email/email.service.ts

```diff
@@ -1,17 +1,58 @@
 import { Injectable } from '@nestjs/common';
 import { SqsMessageHandler, SqsConsumerEventHandler } from '@ssut/nestjs-sqs';
 import * as ses from 'node-ses';
+import { validate } from 'class-validator';
+import { mailbody } from './validator/CustomEmailValidator_server';
+import { Validate, Length, IsEmail, IsArray, ValidateNested } from 'class-validator';
+import { CustomEmailArray } from './validator/CustomEmailArray';
 
 @Injectable()
 export class EmailService {
   private readonly client = ses.createClient({} as any);
+  @IsEmail({ message: 'You need a proper email address' })
```

Comment:
It looks like you should have these fields in a model instead of in the service itself.

Reply:
I will move it to CustomEmailValidator_client.ts in validator.